eng/pipelines/templates/scripts/docsms-allowlist/js-allowlist.txt

```diff
@@ -0,0 +1,7 @@
+^ci-configs/
+^docs-ref-autogen/
+^docs-ref-mapping/
+^docs-ref-services/
+^metadata/
+^preview-packages/docs-ref-autogen/
+^preview-packages/docs-ref-mapping/
```

Comment:
This should also include `^previous-packages/docs-ref-autogen` and `^previous-packages/docs-ref-mapping`

Reply:
Same as above